qml.qrc

```diff
@@ -100,11 +100,20 @@
         <file>lang/flags/bangladesh.png</file>
         <file>lang/flags/brazil.png</file>
         <file>lang/flags/china.png</file>
+        <file>lang/flags/croatia.png</file>
+        <file>lang/flags/esperanto.png</file>
+        <file>lang/flags/finland.png</file>
+        <file>lang/flags/france.png</file>
         <file>lang/flags/german.png</file>
         <file>lang/flags/india.png</file>
+        <file>lang/flags/italy.png</file>
```

Comment:
I didn't put it because there is already a pull request adding the link to indonesia.png (PR #503).